### ISSUE TITLE
Missing '--recursive or -rc' flag in the new release.

### ISSUE THREAD

**Issue by asheux** [timothycrosley/isort#1263]
Getting this error when I use ```--recursive``` flag. Apparently, it's missing in the new release.

- isort v4.3.21 woked perfectly
```-rc, --recursive      Recursively look for Python files of which to sort imports```

![rc](https://user-images.githubusercontent.com/22955146/86524355-5d62bf80-be82-11ea-8ce1-e6b2b929db43.png)

**Comment by timothycrosley** [comment]
That's correct. https://github.com/timothycrosley/isort/blob/develop/CHANGELOG.md#500-penny---july-4-2020 The --recurisive option doesn't mean anything any more as directories passed in are automatically recursively sorted - in line with other popular formatters

**Comment by asheux** [comment]
@timothycrosley , Thanks for the clarification. Also, I tried to run isort using the new recursive command, ```isort .``` along with other flags but the command seems to not work. But when I run it without the additional flags, it works perfectly. Is that the intended behaviour?

Here's the command;
``` isort . --multi-line=3 --trailing-comma --force-grid-wrap=0 --combine-as --line-width 88 --check-only --thirdparty fastapi --thirdparty fastapi --thirdparty pydantic --thirdparty starlette```

And heres my output:
![isort_command](https://user-images.githubusercontent.com/22955146/86525737-c868c200-be93-11ea-9add-55e2cee68694.png)

**Comment by timothycrosley** [comment]
Your output implies that it's working but finding errors (note that `--check-only` means isort wont make any changes just let you know if it would have if not in check-only mode). To get more info, you can add `--diff` which will let you know what changes isort wants to make with your given config. I went ahead and cloned the fast api and used the given command with isort 5.0.0 and it only identified one incorrectly sorted file, adding `--diff` I got: 

```python3
(isort-5f3Bw38m-py3.8) timothy@pop-os:~/Projects/fastapi/docs_src$ isort . --multi-line=3 --trailing-comma --force-grid-wrap=0 --combine-as --line-width 88 --check-only --thirdparty fastapi --thirdparty fastapi --thirdparty pydantic --thirdparty starlette --diff
ERROR: /home/timothy/Projects/fastapi/docs_src/nosql_databases/tutorial001.py Imports are incorrectly sorted and/or formatted.
--- /home/timothy/Projects/fastapi/docs_src/nosql_databases/tutorial001.py:before       2020-07-04 21:53:12.800632
+++ /home/timothy/Projects/fastapi/docs_src/nosql_databases/tutorial001.py:after        2020-07-04 21:54:16.780226
@@ -1,11 +1,10 @@
 from typing import Optional
-
-from fastapi import FastAPI
-from pydantic import BaseModel
 
 from couchbase import LOCKMODE_WAIT
 from couchbase.bucket import Bucket
 from couchbase.cluster import Cluster, PasswordAuthenticator
+from fastapi import FastAPI
+from pydantic import BaseModel
 
 USERPROFILE_DOC_TYPE = "userprofile"
```
Which looks correct to me (unless the intention here was to make couchbase a first party?)

**Comment by asheux** [comment]
Yes. I see my mistake, my initial goal was to format without checking. Therefore, I don't need the ```--check-only``` flag in my command. Thanks.

**Comment by timothycrosley** [comment]
No worries! Just glad I can help!

**Comment by asheux** [comment]
Quick question, how could I check recursively for python files by specifying the folder as an argument to the command? For example; ```isort --check-only folder1 folder2 folder3```.

**Comment by timothycrosley** [comment]
That's exactly right! `isort --check-only folder1 folder2 folder3` should check recursively all 3 folders for any Python file.

**Comment by asheux** [comment]
I actually ran the same command as I posted in the comment with the specific folders and it did the check. Without the dot.

**Comment by eht16** [comment]
Maybe it's worth to add back `--recursive` as a no-op and print a deprecation warning instead?
I guess the current behaviour will break quite some CI setups where `--recursive` was used.

**Comment by Spiderpig86** [comment]
+1 on breaking CIs. Although it really is a small fix, but would be a nice gesture to warn first.

**Comment by timothycrosley** [comment]
Thanks for reporting! This is fixed in the just release 5.0.4 patch release. isort will warn people if the option is encountered and point to the upgrade guide: https://timothycrosley.github.io/isort/docs/upgrade_guides/5.0.0/

**Comment by JamesHutchisonCarta** [comment]
Is it intentional new behavior that this walks down "hidden" paths (example: `.venv`) now?
